File: crates/auto-lang/test/a2j/008_method/method.wrong.js
class Point {
    constructor(x, y) {
        this.x = x;
        this.y = y;
    }

    modulus() {
        this.x * this.x + this.y * this.y;
    }
}

function main() {
    console.log("Method defined");
}

main();

File: crates/auto-lang/test/a2p/008_method/method.wrong.py
class Point:
    def __init__(self, x: Any, y: Any):
        self.x = x
        self.y = y

    def modulus(self):
        self.x * self.x + self.y * self.y

def main():
    print("Method defined")

if __name__ == "__main__":
    main()
